packages/fixtures/src/events.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  BasicElementAction,
+  BlockAction,
+  BlockButtonAction,
+  BlockElementAction,
+  ButtonAction,
+  SlashCommand,
+} from '@slack/bolt';
+
+const token = 'TOKEN';
+const response_url = 'https://fake.slack/response_url';
+const trigger_id = 'TRIGGER_ID';
+const api_app_id = 'API_APP_ID';
+const user = {
+  id: 'UUSERID',
+  name: 'USER',
+};
+const channel = {
+  id: 'CCHANNELID',
+  name: 'channel',
+};
+const team = {
+  id: 'TTEAMID',
+  domain: 'team-domain',
+};
+
+export const slashCommand = (
+  command: string,
+  options: Partial<SlashCommand> = {},
+): SlashCommand => ({
+  command,
+  text: '',
```

Comment:
Yeah I'm into that.

Reply:
I went with simplicity first. Didn't want to write all the things to detect which optional params were passed in based on type ya know?

Cause you can do `event.slashCommand("/command", { text: "A Slash Command" })` which is the same.

I didn't really want to have more than one optional argument, especially when the last one was just, override anything.

CA: 2

Reply:
Yeah, fair, that's cool. options maybe is a bit overloaded in that case, might be worth doc'ing. with this same implementation, someone could do this

```ts
event.slashCommand("/command", { command: "/justkiddingnewcommand" })
````

which is a bit weird.

Reply:
☝️ facts. Not sure what you mean by "Might be worth doc'ing"? Which part? 

I don't really want to document all possible fields, starting to doc one is a slippery slope.

Not without automation anyway.

Reply:
yeah, not all possible fields. maybe an example in the readme just showing that any typical fields can be easily overridden via the "options" argument, since options is options + overrides in this case, maybe even more overrides. It more or less says this in the readme though, so it's probably fine, but might be worth an example. either way this is fine as is